runelite-client/src/main/java/net/runelite/client/plugins/party/PartyPlugin.java

```diff
@@ -128,6 +128,7 @@
 
 	private int lastHp, lastPray;
 	private boolean hotkeyDown, doSync;
+	private boolean sendAlert = false;
```

Comment:
booleans default to false, so this initialization isn't needed.
```suggestion
	private boolean sendAlert;
```